back-end/API/tests/test_user.py

```
@@ -116,196 +131,191 @@ def test_resetPasswordNormal(client):
     
     #Mettre l'utilisateur en admin
-    data2 = {
-        "email": "utilisateur1@gmail.com"
-    }
-    response1 = client.put('/user/makeAdmin', json=data2)
+    data2 = {}
+    response1 = client.put('/user/toggleAdmin/7', json=data2)
+    assert response1.status_code == 200
+    response = client.get('/user/7')
+    assert response.status_code == 200
+    assert response.json['isModerator'] == True
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `True`; use `response.json['isModerator']:` for truth checks

Replace with `response.json['isModerator']`

(E712)